index.bs

```diff
@@ -1490,16 +1490,29 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
 1. Let |credTypesAndPubKeyAlgs| be a new [=list=] whose [=list/items=] are pairs of {{PublicKeyCredentialType}} and
     a {{COSEAlgorithmIdentifier}}.
 
-1. [=list/For each=] |current| of <code>|options|.{{PublicKeyCredentialCreationOptions/pubKeyCredParams}}</code>:
-
-    1. If <code>|current|.{{PublicKeyCredentialParameters/type}}</code> does not contain a {{PublicKeyCredentialType}} supported
-        by this implementation, then [=continue=].
-    1. Let |alg| be <code>|current|.{{PublicKeyCredentialParameters/alg}}</code>.
-    1. [=list/Append=] the pair of <code>|current|.{{PublicKeyCredentialParameters/type}}</code> and |alg| to
-        |credTypesAndPubKeyAlgs|.
-
-1. If |credTypesAndPubKeyAlgs| [=list/is empty=] and <code>|options|.{{PublicKeyCredentialCreationOptions/pubKeyCredParams}}</code>
-    [=list/is not empty=], return a {{DOMException}} whose name is "{{NotSupportedError}}", and terminate this algorithm.
+1. If <code>|options|.{{PublicKeyCredentialCreationOptions/pubKeyCredParams}}</code>'s [=list/size=]
+    <dl class="switch">
+        :   is zero
+        ::  Let |defaultPublicKeyAlgorithm| be a {{COSEAlgorithmIdentifier}} with one of the following values at
+            the discretion of the [=client platform=]:
+            *  <code>-7</code> ("ES256").
+            *  <code>-257</code> ("RS256").
+
+            [=list/Append=] the pair of {{PublicKeyCredentialType}} with value {{public-key}} and a
+            {{COSEAlgorithmIdentifier}} with value |defaultPublicKeyAlgorithm| to |credTypesAndPubKeyAlgs|.
+
+        :   is non-zero
+        ::  [=list/For each=] |current| of <code>|options|.{{PublicKeyCredentialCreationOptions/pubKeyCredParams}}</code>:
+
+            1. If <code>|current|.{{PublicKeyCredentialParameters/type}}</code> does not contain a {{PublicKeyCredentialType}} supported
+                by this implementation, then [=continue=].
+            1. Let |alg| be <code>|current|.{{PublicKeyCredentialParameters/alg}}</code>.
+            1. [=list/Append=] the pair of <code>|current|.{{PublicKeyCredentialParameters/type}}</code> and |alg| to
+                |credTypesAndPubKeyAlgs|.
+
+            If |credTypesAndPubKeyAlgs| [=list/is empty=], return a {{DOMException}} whose name is
+               "{{NotSupportedError}}", and terminate this algorithm.
+    </dl>
```

Comment:
Not asking that client chooses one of the algorithm at its descretion. Would like the list be of two elements. 

We just need to add one more step in existing flow. `If <code>|options|.{{PublicKeyCredentialCreationOptions/pubKeyCredParams}}</code> is empty, append <code>-7</code> ("ES256") AND <code>-257</code> ("RS256") to the list.`

Reply:
Oh I get you now. Updated the PR to append both `-7` and `-257` to the list.